src/applications/vaos/redux/api/vaosApi.js

```diff
@@ -0,0 +1,27 @@
+import { createApi } from '@reduxjs/toolkit/query/react';
+import { apiRequestWithUrl } from 'applications/vaos/services/utils';
+
+export const vaosApi = createApi({
+  reducerPath: 'appointmentApi',
+  baseQuery: () => ({ data: null }),
+  endpoints: builder => ({
+    getReferralById: builder.query({
+      async queryFn(referralId) {
+        try {
+          return await apiRequestWithUrl(
+            `/vaos/v2/epsApi/referrals/${referralId}`,
+            {
+              method: 'GET',
```

Comment:
nit: I don't think this is needed since the default method is `GET` as defined [here](https://github.com/department-of-veterans-affairs/vets-website/blob/19bf477347f482d80c9a1b6314460c71bb261183/src/platform/utilities/api/index.js#L118). Is there any reason this is explicitly set here?